### PR TITLE
Add note

### PR DESCRIPTION
**Pull request by ruthryi**
Fixes https://github.com/wso2-enterprise/wso2-apim-internal/issues/33#issuecomment-1134182038

### CODE REVIEW

File: en/docs/reference/connectors/kafka-connector/setting-up-kafka.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ Copy the following additional client libraries to the `<MI_HOME>/lib` directory
 * [common-utils-5.4.0.jar](https://mvnrepository.com/artifact/io.confluent/common-utils/5.4.0)
 * [kafka-avro-serializer-5.3.0.jar](https://mvnrepository.com/artifact/io.confluent/kafka-avro-serializer/5.3.0)
 * [kafka-schema-registry-client-5.3.0.jar](https://mvnrepository.com/artifact/io.confluent/kafka-schema-registry-client/5.3.0)
+   
+!!! Note "Library versions"
+    Note that the libraries mentioned above are the versions **tested** with the connector.
```
Comment (Nirothipan) [inline en/docs/reference/connectors/kafka-connector/setting-up-kafka.md:25]: after following the steps only, they will see this note. We need to tell the user before hand that following the versions we've tested, giving a hint that they can use whatever they want.

Comment (ruthryi) [reply en/docs/reference/connectors/kafka-connector/setting-up-kafka.md]: Done

Comment (Nirothipan) [reply en/docs/reference/connectors/kafka-connector/setting-up-kafka.md]: Shall we further it to say, you can use any compatible version after local testing

File: en/docs/reference/connectors/kafka-connector/setting-up-kafka.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ To use the Kafka connector, download and install [Apache Kafka](http://kafka.apa
 
 > **Note**: The recommended version is [Kafka_2.11-2.2.1](https://www.apache.org/dyn/closer.cgi?path=/kafka/1.0.0/kafka_2.11-2.2.1.tgz). For all available versions of Kafka that you can download, see https://kafka.apache.org/downloads. The recommended Java version is 1.8.
 
+!!! Note "Library versions"
+    Note that the libraries mentioned below are the versions **tested** with the connector.
```
Comment (samgnaniah) [inline en/docs/reference/connectors/kafka-connector/setting-up-kafka.md:8]:
```suggestion
    Note that the libraries mentioned below are the versions **tested** with the connector. You can use different versions of the libraries if you test them locally.
```

**Pull request merged**